src/modules/songs/components/SongTags/SongTags.tsx

```
@@ -0,0 +1,114 @@
+    );
+  };
+
+  if (songTagsQueryError) {
+    toast.error(
+      `Could not the tags for this song: ${songTagsQueryError.message}`,
+    );
+    return null;
+  }
+
+  if (isSongTagsQueryLoading) {
+    return <HStack className="gap-2"></HStack>;
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)